### ISSUE TITLE
Better treatment of disconnected chunks of data in entity graph

### ISSUE THREAD

**Issue by manuelma** [Spine-project/Spine-Toolbox#1593]
The layout generator in the graph view considers islands as part of the same system, buy maybe it's better to generate layouts independently for each island and then let them be superposed in the final result. Below is a comparison of the current situation with what could be:

![spine_case11 pdf](https://user-images.githubusercontent.com/11472756/140379719-cd1bd8ce-d689-4424-b5f1-cee166cf1e5e.png)
![case11d](https://user-images.githubusercontent.com/11472756/140379733-b00fe69d-92f0-4757-bc97-db1bd00b05f4.png)

**Comment by jkiviluo** [comment]
I don't quite get it. Are there no common nodes or units between these two grids? If that's the case, shouldn't they then be plotted with an 'air-gap' instead of crossing lines?

**Comment by manuelma** [comment]
Yes, I think so. I guess the ideal is the second picture but without the overlapping, i.e., each grid should be put on one side of the canvas.
